config/environments/production.rb

```diff
@@ -76,4 +76,13 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.paperclip_defaults = {
+    storage: :s3,
+    s3_credentials: {
+      bucket: "cardapp",
+      access_key_id: ENV['s3_key'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.